### DISCUSSION TITLE
Sliders for Servername/Tribe-Group on Dashboard

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#140] [Suggestions]
A slider or something to display longer names better. 
![grafik](https://user-images.githubusercontent.com/99832504/160402014-8e34d62a-cdb7-4a9d-bc3a-62f92b25979f.png)

I know you can hover over but so many different servers to hover over CAN be a pain :D

**Comment by mvaessen** [comment]
Hello and thanks for your suggestion!

This is mainly done to accommodate mobile devices. The current dashboard is rather old and limited. It will be replaced with a modular system later this year, that will have better mobile & desktop support.
